examples/gallery/images/rgb_image.py

```diff
@@ -28,7 +28,7 @@
     # Subset to area of Lāhainā in EPSG:32604 coordinates
     image = img.rio.clip_box(minx=738000, maxx=755000, miny=2300000, maxy=2318000)
     image = image.load()  # Force loading the DataArray into memory
-image
+print(image)
```

Comment:
This now shows the plaintext version of the xarray.DataArray instead of the repr - https://pygmt-dev--2825.org.readthedocs.build/en/2825/gallery/images/rgb_image.html#sphx-glr-gallery-images-rgb-image-py. Maybe try 
```suggestion
repr(image)
```

Reply:
It looks worse now.

Reply:
The fancy representation on this page (https://www.pygmt.org/dev/gallery/images/rgb_image.html#sphx-glr-gallery-images-rgb-image-py) is done by the xarray.DataArray's `_repr_html_` method (https://ipython.readthedocs.io/en/stable/config/integrating.html#rich-display), which is different the `repr` function.

Our options:

1. Use `print(image)`
2. Completely remove `imgae`
3. Ignore the error `image  # noqa: B018`

Reply:
Let's try Option 3 (`noqa`). Doesn't look nice, but at least the repr_html shows up.

![image](https://github.com/GenericMappingTools/pygmt/assets/23487320/3c872177-5863-4044-a0fa-d8b21d50c994)